docker/Dockerfile

```
@@ -0,0 +1,20 @@
+FROM golang:1.21 as builder
+
+WORKDIR /build
+COPY go.mod go.mod
+COPY go.sum go.sum
+RUN go mod download
+COPY pkg/ pkg/
+COPY templates/ templates/
+COPY main.go main.go
+RUN ls -lt 
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o zuul-mqtt-matrix-bridge main.go
```

```suggestion
RUN go build -a -o zuul-mqtt-matrix-bridge main.go
```

i'd do it like that, that way the Dockerfile can be used to build images for varying architectures.

---

agree